src/components/UploadField.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { withFirebase } from 'react-redux-firebase';
+import Dropzone from 'react-dropzone'; // For Dropzone reference check https://github.com/react-dropzone/react-dropzone
+
+@withFirebase
+class UploadField extends Component {
+  state = {
+    isLoading: false,
+    files: [],
+  }
+
+  constructor(props) {
+    super(props);
+
+    // Path in the Storage where the files will get saves
+    this.storagePath = this.props.storagePath;
+
+    // Path in the db where the files will get saved
```

Comment:
comments dont make sense together. `files will get saved` in both Storage and db? Surely the db contains a reference to the Storage file?

Reply:
yes, how doesn't it make sense? besides the typo?

Reply:
How can the files be saved in both Storage and db? What does that mean?

Reply:
I see, my bad. I should have said ` // Path in the db where the reference to the file will get saved`

That makes sense?